### PR TITLE
Simplify cell columns on grid component

### PR DESCRIPTION
**Pull request by nkpgardose**
- group default and device column classes on each device media queries

### CODE REVIEW

**Comment by Garbee** [comment]
LGTM.

Manually checked out commit and the build passes locally. So, looks like the Drone system having a small glitch?

**Comment by nkpgardose** [comment]
it looks like it. Build also passes locally on my side.

**Comment by addyosmani** [comment]
Hmm. Seeing it fail in Drone here:

```
[03:56:16] Starting 'styles:gen'...
[03:56:16] 'styles:gen' errored after 1.92 ms
[03:56:16] Error: ENOENT, no such file or directory './dist/material.min.css.template'
    at Object.fs.openSync (fs.js:439:18)
    at Object.fs.readFileSync (fs.js:290:15)
    at Gulp.<anonymous> (/var/cache/drone/src/github.com/google/material-design-lite/gulpfile.js:712:20)
    at module.exports (/cache/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/cache/node_modules/gulp/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/cache/node_modules/gulp/node_modules/orchestrator/index.js:214:10)
    at /cache/node_modules/gulp/node_modules/orchestrator/index.js:279:18
    at finish (/cache/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:21:8)
    at /cache/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:52:4
    at f (/cache/node_modules/gulp/node_modules/orchestrator/node_modules/end-of-stream/node_modules/once/once.js:17:25)
[03:56:16] 'all' errored after 5.43 s
[03:56:16] Error: [object Object]
    at formatError (/usr/local/lib/node_modules/gulp/bin/gulp.js:169:10)
    at Gulp.<anonymous> (/usr/local/lib/node_modules/gulp/bin/gulp.js:195:15)
    at Gulp.emit (events.js:117:20)
    at Gulp.Orchestrator._emitTaskDone (/cache/node_modules/gulp/node_modules/orchestrator/index.js:264:8)
    at /cache/node_modules/gulp/node_modules/orchestrator/index.js:275:23
    at finish (/cache/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:21:8)
    at cb (/cache/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:29:3)
    at finish (/cache/node_modules/run-sequence/index.js:53:5)
    at Gulp.onError (/cache/node_modules/run-sequence/index.js:60:4)
    at Gulp.emit (events.js:117:20)
[03:56:16] 'styles' all files 475.82 kB
[03:56:16] Finished 'styletemplates' after 5.42 s
```

**Comment by nkpgardose** [comment]
@addyosmani Hmm. I'll check on that.

**Comment by surma** [comment]
LGTM, thanks @nkpgardose!

File: src/grid/_grid.scss [hunk 1]
```diff
@@ -87,10 +87,7 @@
 
   // Define partial sizes for columnNumber < totalColumns.
   @for $i from 1 through ($grid-phone-columns - 1) {
-    .mdl-cell--#{$i}-col {
-      @include partial-size($i, $grid-phone-columns, $grid-phone-gutter);
-    }
-
+    .mdl-cell--#{$i}-col,
     .mdl-cell--#{$i}-col-phone.mdl-cell--#{$i}-col-phone {
```
Comment (nkpgardose) [inline src/grid/_grid.scss]: Just noticed multiple selector declaration. Any reason behind this? Or I can change this into ```.mdl-cell-#{i}-col-phone``` only.

Comment (Garbee) [inline src/grid/_grid.scss]: Leave the multiple declarations. It is to help address specificity issues.

Comment (nkpgardose) [inline src/grid/_grid.scss]: :+1:  TIL! @Garbee

**Comment by nkpgardose** [comment]
@surma alright. no prob!
GIF

**Comment by addyosmani** [comment]
Landing for 1.0.3. Thanks for addressing the comments!

**Pull request closed without merging**